spring-batch-infrastructure/src/test/java/org/springframework/batch/item/file/builder/FlatFileItemReaderBuilderTests.java

```diff
@@ -85,6 +86,44 @@ public void testSimpleDelimited() throws Exception {
 		assertNull(reader.read());
 	}
 
+	@Test
+	public void testSimpleDelimitedWithWhitespaceCharacter() throws Exception {
+		FlatFileItemReader<Foo> reader = new FlatFileItemReaderBuilder<Foo>()
+				.name("fooReader")
+				.resource(getResource("1 2 3"))
+				.delimited()
+				.delimiter(" ")
+				.names(new String[] {"first", "second", "third"})
+				.targetType(Foo.class)
+				.build();
+
+		reader.open(new ExecutionContext());
+		Foo item = reader.read();
+		assertEquals(1, item.getFirst());
+		assertEquals(2, item.getSecond());
+		assertEquals("3", item.getThird());
+		assertNull(reader.read());
+	}
+
+	@Test
+	public void testSimpleDelimitedWithTabCharacter() throws Exception {
+		FlatFileItemReader<Foo> reader = new FlatFileItemReaderBuilder<Foo>()
+				.name("fooReader")
+				.resource(getResource("1	2	3"))
```

Comment:
Please replace the tab with `\t` so that it is explicit.`

Reply:
I agree! Fixed.